dashboard/app/controllers/api_controller.rb

```diff
@@ -200,7 +200,7 @@ def user_progress_for_stage
       end
     end
 
-    if level.finishable?
```

Comment:
Is the right fix? Do we not want to `slog` when we don't have a `script_level`? Naively, I'd expect the fix to be changing `script_level.id` to `script_level.try(:id)`.